firebase_admin/_utils.py

```diff
@@ -59,6 +59,25 @@
 }
 
 
+_RPC_CODE_TO_ERROR_CODE = {
```

Comment:
Nit: May be mention https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto in a comment so we know where these mappings come from.

Reply:
done